package.json

```diff
@@ -124,14 +124,14 @@
     "imports-loader": "0.7.1",
     "jetifier": "1.6.4",
     "metro-react-native-babel-preset": "0.56.0",
-    "node-sass": "4.13.1",
+    "node-sass": "4.14.1",
     "string-replace-loader": "2.1.1",
     "style-loader": "0.19.0",
     "unorm": "1.6.0",
-    "webpack": "4.27.1",
+    "webpack": "4.43.0",
```

Comment:
This `closes #6120`. See: https://github.com/jitsi/jitsi-meet/pull/6120/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R130